### PR TITLE
feat: add outfits

### PR DESCRIPTION
**Pull request by pedrotambo**

### CODE REVIEW

File: src/platform/outfits/outfits.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+export type Outfits = {
+  outfits: {
+    slot: number
+    outfit: Outfit
+  }[]
+  namesForExtraSlots: string[]
+}
+
+/**
+ * Avatar
```
Comment (marianogoldman) [inline src/platform/outfits/outfits.ts:72]:
```suggestion
 * Outfits
```

File: src/platform/outfits/outfits.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    type: 'object',
+    required: ['outfits', 'namesForExtraSlots'],
+    properties: {
+      outfits: {
+        type: 'array',
+        items: {
+          type: 'object',
+          required: ['slot', 'outfit'],
+          properties: {
+            slot: {
```
Comment (marianogoldman) [inline src/platform/outfits/outfits.ts:86]: Perhaps make slot id unique?

File: src/platform/outfits/outfits.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+              type: 'number'
+            },
+            outfit: Outfit.schema
+          }
+        }
+      },
+      namesForExtraSlots: {
+        type: 'array',
+        items: {
+          type: 'string'
```
Comment (marianogoldman) [inline src/platform/outfits/outfits.ts:96]: Do not allow duplicates.

File: src/platform/outfits/outfits.ts (hunk 2 above)
Comment (pedrotambo) [reply src/platform/outfits/outfits.ts:86]: Is not as easy to set `uniqueItems = true` as the slot is within an object but not an array of slots. That's validated in `content-validator`

File: src/platform/outfits/outfits.ts (hunk 3 above)
Comment (pedrotambo) [reply src/platform/outfits/outfits.ts:96]: will change

Comment (pedrotambo) [reply src/platform/outfits/outfits.ts:97]: added

**Review by marianogoldman — APPROVED** [review]

**Pull request merged**